### ISSUE TITLE
comment-word-blacklist - warns only if comment content == string value

### ISSUE THREAD

**Issue by dryoma** [stylelint/stylelint#1643]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Not sure if this is a bug or intended behavior. Yet, stylelint yields a warning only if a comment *consists only of* one of the words from a blacklist, not just has them. It of cource could be achieved with `["/prohibited-word/"]` regex, but from the first glance it seems natural for this rule to warn if a string from the value array is *among* the comment contents.

> Which rule, if any, is this issue related to?

`comment-word-blacklist`

> What CSS is needed to reproduce this issue?

```css
/* prohibited-word allowed-word*/
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "comment-word-blacklist": ["prohibited-word"]
  }
}
```

> Which version of stylelint are you using?

`6.9.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

via gulp PostCSS plugin.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Don't think so.

> What did you expect to happen?

`Unexpected word matching pattern "ignored-word" (comment-word-blacklist)`

> What actually happened (e.g. what warnings or errors you are getting)?

No warning.

**Comment by jeddy3** [comment]
@dryoma Can you try `7.0.1` please, as I think [this might of been](https://github.com/stylelint/stylelint/pull/1463) addressed there.

**Comment by dryoma** [comment]
Yup, it works in v7, thanks.

stylelint sure moves fast! It was just the day after yesterday that I updated to 6.9 and tried that rule.

**Comment by jeddy3** [comment]
> stylelint sure moves fast!

Yep, it does. Although, the last major release for 4 months ago. This new release is significant in that a lot of work has gone into making development on the linter more sustainable i.e. putting in place systems to empower the community to build plugins and processors, rather than have these volatile features in core. Hopefully this will slow down the pace of stylelint releases (especially major ones) even further. I think most of the contributors are looking forward to a little breather as well now that 7.0.0 is out of the door :)
